pkg/machine/ocipull/pull.go

```diff
@@ -26,8 +27,21 @@ type PullOptions struct {
 	Quiet bool
 }
 
+var (
+	noSignaturePolicy string = `{
+    "default": [
+        {
+            "type": "insecureAcceptAnything"
+        }
+    ]
+}`
```

Comment:
I would remove all the spaces/newlines from the json string here. I know it is arguably better for readability but I think we should safe the bytes. It is not much but in the bigger picture I think every byte matters given how much our size is currently.